store/storetest/user_store.go

```diff
@@ -2336,7 +2336,7 @@ func testUserUnreadCount(t *testing.T, ss store.Store) {
 	_, nErr = ss.Post().Save(&p3)
 	require.NoError(t, nErr)
 
-	nErr = ss.Channel().IncrementMentionCount(c2.Id, u2.Id, false, false)
+	nErr = ss.Channel().IncrementMentionCount(c2.Id, []string{u2.Id}, false, false)
```

Comment:
I don't see tests with more than one id (they should work fine, but still, would be nice to have).

Also, just as a sanity check, it would be nice to have a test with the same id repeated. Looking at the code it should be solved fine, but seems a nice to have too.

Reply:
Yep, good point. I am just waiting for Jesse to return and merge the other PR on which it's blocked.

Reply:
I've resolved the merge conflicts and should auto-merge https://github.com/mattermost/mattermost-server/pull/19523 shortly.

Reply:
Tests added!

Reply:
Maybe I am missing something, but I don't see how the new tests cover the two cases I noted (more than one id, and repeated id).

Reply:
PTAL :)